setup/azurecmdfiles.wxi

```diff
@@ -3239,6 +3239,62 @@
                             <File Id="fil8A60FFA36529EBE3E1D38387A2EF0875" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Relay\System.Net.Http.Primitives.dll" />
                         </Component>
                     </Directory>
+                    <Directory Id="dirB083EC6D83AC4367D33EB257B3A8A832" Name="AzureRM.Reservations">
+                        <Component Id="cmp325C5D7E7BFCC111B2335A376E2779D4" Guid="*">
+                            <File Id="filB827DC440AC145244BDA78F4A4494806" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\AzureRM.Reservations.psd1" />
+                        </Component>
+                        <Component Id="cmp4320ADA5285B6EB4FDB2A625953B4BFD" Guid="*">
+                            <File Id="filDBD6A7D1882D4D599FA965CEDBE72A90" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\Hyak.Common.dll" />
+                        </Component>
+                        <Component Id="cmp8EA9AE56244697DD7B80808162AEC7BE" Guid="*">
+                            <File Id="fil42810BE4C46304EFADE0D35134142C76" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\Microsoft.ApplicationInsights.dll" />
+                        </Component>
+                        <Component Id="cmp96A65C72FE898B2763E8AF8ECC400499" Guid="*">
+                            <File Id="filEB747F767B8EC1DD765F5A73459BBB21" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\Microsoft.Azure.Commands.Common.Authentication.Abstractions.dll" />
+                        </Component>
+                        <Component Id="cmpA0A77CCCC866B08C257B1786CB2ABF21" Guid="*">
+                            <File Id="filE1CB1F1556BFCD7312E8B764060C3EA1" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\Microsoft.Azure.Commands.Common.Authentication.dll" />
+                        </Component>
+                        <Component Id="cmp6DF24F4A515BA29743DBC75F1085247F" Guid="*">
+                            <File Id="filE1E5B68E35D0B658BCB8A5473C6113C1" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\Microsoft.Azure.Commands.Reservations.dll" />
+                        </Component>
+                        <Component Id="cmp12D613E47A5C3F6A9FC1D1755563A38B" Guid="*">
+                            <File Id="filEF613C086E8C46888B119F7DB456AEC2" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Reservations\Microsoft.Azure.Commands.Reservations.dll-Help.xml" />
```

Comment:
This is causing merge conflicts right now.  You will need to pull the latest, cleam, rebuild, and regenerate this file

Reply:
done. By the way, I looked at the merge conflict and saw that it's just my commit adding few more lines to wxi, it does not delete anything in base branch. Not sure why I got merge conflict.

Reply:
@markcowl There is still merge conflict but as I said, it' just addition from our branch. Can you merge this change?

Reply:
No, you will need to pull the latest from the branch, resolve conflicts, and regenerate.